frontend/packages/kubevirt-plugin/src/components/modals/vm-boot-order-modal/components/dnd-list/dnd-list.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { DataList } from '@patternfly/react-core';
+import { PficonDragdropIcon } from '@patternfly/react-icons';
+import { DndProvider } from 'react-dnd';
+import HTML5Backend from 'react-dnd-html5-backend';
+import { DNDRowAddItem } from './dnd-row-add-item';
+import { DNDRow } from './dnd-row';
+
+import './dnd-list.scss';
+
+export const DNDList: React.FC<DNDListProps> = ({
```

Comment:
IMO, I don't think this component is really necessary. We could just use the body inplace, because the number of props is not that much different from the number of lines of the body

Reply:
Shouldn't this be reusable for other components ?

Reply:
I think one can put this together in the place of need very easily. It is just that this component is heavily customized (props) and one could just use the plain components instead